src/mdastToDocxModel.ts

```
@@ -242,14 +263,16 @@ export function mdastToDocxModel(root: Root, _style: Style, _options: Options):
-    
+
     for (const node of nodes) {
       switch (node.type) {
         case "text":
           pushTextWithUnderline((node as Text).value);
           break;
         case "emphasis":
-          const emphasisChildren = processInlineNodes((node as Emphasis).children);
+          const emphasisChildren = processInlineNodes(
+            (node as Emphasis).children,
+          );
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)